primitives/core/src/sr25519.rs

```diff
@@ -532,19 +532,22 @@ impl TraitPair for Pair {
 	/// Verify a signature on a message. Returns true if the signature is good.
 	fn verify<M: AsRef<[u8]>>(sig: &Self::Signature, message: M, pubkey: &Self::Public) -> bool {
 		Self::verify_weak(&sig.0[..], message, pubkey)
+
 	}
 
 	/// Verify a signature on a message. Returns true if the signature is good.
 	fn verify_weak<P: AsRef<[u8]>, M: AsRef<[u8]>>(sig: &[u8], message: M, pubkey: P) -> bool {
-		// Match both schnorrkel 0.1.1 and 0.8.0+ signatures, supporting both wallets
-		// that have not been upgraded and those that have. To swap to 0.8.0 only,
-		// create `schnorrkel::Signature` and pass that into `verify_simple`
-		match PublicKey::from_bytes(pubkey.as_ref()) {
-			Ok(pk) => pk.verify_simple_preaudit_deprecated(
-				SIGNING_CTX, message.as_ref(), &sig,
-			).is_ok(),
-			Err(_) => false,
-		}
+		let signature = match schnorrkel::Signature::from_bytes(sig) {
```

Comment:
Maybe we should try to sync kusama with this, not sure if there's txs signed with schnorrkel 0.1.1 that we won't be able to verify anymore. cc @burdges

Reply:
Or BABE blocks for that matter.

Reply:
> Maybe we should try to sync kusama with this, not sure if there's txs signed with schnorrkel 0.1.1 that we won't be able to verify anymore. cc @burdges

Yep, trying to do it right now

@burdges said something that 1.1 were only used on Alexander

Reply:
We never supported VRFs from 0.1.1 because BABE did not work until after the audit.  ;)

Just send schnorrkel a PR reverting https://github.com/w3f/schnorrkel/commit/8f3706591dcf0648086b5dfb4bd832970483ac03 if you really need it, but if these exist then we maybe we could keep that pain isolated to kusama, and disable the `preaudit_deprecated` feature for polkadot.

Reply:
There will be old txs (old blocks) signed with 0.1.1 - the JS side only moved to 0.8.5 in Jan once Alex was dropped.